src/Notepads/Controls/Settings/TextAndEditorSettings.xaml.cs

```diff
@@ -60,6 +60,7 @@ public TextAndEditorSettings()
 
             TextWrappingToggle.IsOn = (EditorSettingsService.EditorDefaultTextWrapping == TextWrapping.Wrap);
             HighlightMisspelledWordsToggle.IsOn = EditorSettingsService.IsHighlightMisspelledWordsEnabled;
+            LineHighlighterToggle.IsOn = EditorSettingsService.EditorDefaultLineHighlighterViewState;
```

Comment:
EditorDefaultLineHighlighterViewState is good but let's keep our existing naming convention here by renaming it to "IsLineHighlighterEnabled"